Exosite.cpp

```diff
@@ -94,6 +94,8 @@ boolean Exosite::writeRead(const char* writeString, const char* readString, char
     client->stop();
   #ifdef SL_DRIVER_VERSION
     client->sslConnect(serverName,443);
+  #if defined(ESP8266)
+      client->connect(serverName,443);
```

Comment:
We need to fix the hardcoding of the port number.  But not in this PR.

Reply:
Note that line 97 needs to be changed to:
#elif (ESP8266)